src/Stratis.Bitcoin.Features.PoA/PoANetwork.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NBitcoin;
+using NBitcoin.DataEncoders;
+
+namespace Stratis.Bitcoin.Features.PoA
+{
+    public class PoANetwork : Network
```

Comment:
But we alredy agreed that in order to avoid bumping into each other with sidechains project we would keep a separated network for the development process? 
After PoA is completed we are safe to delete this network and use a new one.

Reply:
Ok, convinced!